webapi/src/main/scala/org/knora/webapi/responders/v1/ValuesResponderV1.scala

```diff
@@ -1939,11 +1946,13 @@ class ValuesResponderV1 extends ResponderV1 {
 
             // Generate a SPARQL update string.
             sparqlUpdate = queries.sparql.v1.txt.createLink(
+                resourceIndex = 0,
```

Comment:
Add a comment explaining that `resourceIndex = 0` because this method isn't used when creating multiple resources.